<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLLongitudinalPETCTAnalysisWidget</class>
 <widget class="QWidget" name="qMRMLLongitudinalPETCTAnalysisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>16</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="ButtonQualitativeAnalysis">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Click here to open the qualitative analysis comparison view</string>
       </property>
       <property name="text">
        <string> Qualitative Analysis</string>
       </property>
       <property name="icon">
        <iconset resource="../qSlicerLongitudinalPETCTModule.qrc">
         <normaloff>:/Icons/qual_icon.png</normaloff>:/Icons/qual_icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string>ButtonGroupAnalysis</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonQuantitativeAnalysis">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Click here to open the quantitative analysis chart view for the computed SUV values</string>
       </property>
       <property name="text">
        <string> Quantitative Analysis</string>
       </property>
       <property name="icon">
        <iconset resource="../qSlicerLongitudinalPETCTModule.qrc">
         <normaloff>:/Icons/quan_icon.png</normaloff>:/Icons/quan_icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <attribute name="buttonGroup">
        <string>ButtonGroupAnalysis</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="LabelInfo">
       <property name="styleSheet">
        <string notr="true">QLabel QToolTip {background-color: #858585; color: #FFF;}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../qSlicerLongitudinalPETCTModule.qrc">:/Icons/help.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Select studies for comparison:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="qMRMLLongitudinalPETCTStudyTableWidget" name="TableStudyAnalysisSelection" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonExportCSV">
       <property name="toolTip">
        <string>Export the calculated SUV values to a Comma-separated values (csv) file </string>
       </property>
       <property name="text">
        <string>Export csv-file </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLLongitudinalPETCTStudyTableWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLLongitudinalPETCTStudyTableWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qSlicerLongitudinalPETCTModule.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="ButtonGroupAnalysis"/>
 </buttongroups>
</ui>
